telcell/celldb/pgdatabase.py

```diff
@@ -0,0 +1,414 @@
+import csv
+import datetime
+import math
+import warnings
+from typing import Optional, List, Iterable, Callable, Tuple
+
+import geopy
+
+from telcell.cell_identity import (
+    Radio,
+    CellIdentity,
+    CellGlobalIdentity,
+    EutranCellGlobalIdentity,
+)
+from . import duplicate_policy
+from .cell_collection import CellCollection, Properties
+from ..data.models import RD_TO_WGS84, WGS84_TO_RD
+from ..geography import Angle
+
+RD_X_RANGE = (7000, 300000)
+RD_Y_RANGE = (289000, 629000)
+
+
+def rd_to_point(x: int, y: int) -> geopy.Point:
+    if not (RD_Y_RANGE[0] <= y <= RD_Y_RANGE[1]) or not (
+        RD_X_RANGE[0] <= x <= RD_X_RANGE[1]
+    ):
+        warnings.warn(
+            f"rijksdriehoek coordinates {x}, {y} outside range: x={RD_X_RANGE}, y={RD_Y_RANGE}"
+        )
+    c = RD_TO_WGS84.transform(x, y)
+    return geopy.Point(longitude=c[0], latitude=c[1])
+
+
+def point_to_rd(point: geopy.Point) -> Tuple[int, int]:
+    return WGS84_TO_RD.transform(point.longitude, point.latitude)
+
+
+def _build_antenna(row):
```

Comment:
typing, en ik vind het gek dat we bij _build_antenna niet de class Antenna teruggeven

Reply:
Typing gefixt, met het andere moeten we inderdaad iets, maar niet in deze PR want dan wordt ie te groot